src/material/core/datetime/date-selection-model.ts

```diff
@@ -0,0 +1,135 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {DateAdapter} from '@angular/material/core';
+import {Subject} from 'rxjs';
+
+export abstract class MatDateSelectionModel<D> {
+  valueChangesSubject = new Subject<void>();
```

Comment:
This should be private and we still need the `valueChanges` `Observable` for the public API. Just when you're calling `.complete()` and `.next()` those methods only exist on the `Subject`

Reply:
Since it's a private property, I won't be able to call this._valueChangesSubject in the derived classes.

Reply:
oh sorry, then you can make it `protected`, the important thing is its not part of the public API

Reply:
Done.